dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/docker/CAContainer.java

```diff
@@ -0,0 +1,372 @@
+package com.ibm.ws.security.acme.docker;
+
+import static junit.framework.Assert.fail;
+
+import java.io.IOException;
+import java.security.cert.X509Certificate;
+import java.util.Map.Entry;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.http.StatusLine;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+
+import com.github.dockerjava.api.model.ContainerNetwork;
+import com.ibm.websphere.simplicity.log.Log;
+import com.ibm.ws.security.acme.utils.AcmeFatUtils;
+
+public class CAContainer extends GenericContainer<BoulderContainer> {
+
+
+	public static final int HTTP_PORT = 5002;
+
+	/** The REST management API port. */
+	public int MANAGEMENT_PORT = 8055;
```

Comment:
Should this be static? Or does each override it?

Reply:
I see you set this in the construct. This field is confusing since it ghosted statically in the child classes. They won't all agree, which could be confusing.